modules/simple_forms_api/spec/services/notification_email_spec.rb

```diff
@@ -12,6 +12,10 @@
         )
         JSON.parse(fixture_path.read)
       end
+      let(:config) do
+        { form_data: data, form_number: 'vba_21_10210',
+          confirmation_number: 'confirmation_number' }
+      end
 
```

Comment:
It would be prudent to also have test coverage for when we provide invalid or nil config parameters as well.